ocaml/typing/persistent_env.mli

```diff
@@ -16,15 +16,18 @@
 
 open Misc
 
+module Impl : sig
+  type t
+end
+
 module Consistbl : module type of struct
-  include Consistbl.Make (Compilation_unit.Name) (Compilation_unit)
+  include Consistbl.Make (Compilation_unit.Name) (Impl)
 end
 
-type error =
+type error = private
```

Comment:
No more than any other, but really they should all be `private` so that we get a warning when a constructor isn't used. Also it would be very weird (though I guess not actually dangerous) for another module to create a value of this type.

Reply:
If there is no extra reason then please stick the same convention as elsewhere

Reply:
Reverted the change.